unstructured/partition/email.py

```diff
@@ -131,21 +166,29 @@ def build_email_metadata(
 ) -> ElementMetadata:
     """Creates an ElementMetadata object from the header information in the email."""
     signature = find_signature(msg)
-
     header_dict = dict(msg.raw_items())
     email_date = header_dict.get("Date")
+
+    def parse_recipients(header_value: Optional[str]) -> Optional[list[str]]:
+        if header_value is not None:
+            return [recipient.strip() for recipient in header_value.split(",")]
+        return None
+
     if email_date is not None:
         email_date = convert_to_iso_8601(email_date)
 
-    sent_from = header_dict.get("From")
-    if sent_from is not None:
-        sent_from = [sender.strip() for sender in sent_from.split(",")]
-
-    sent_to = header_dict.get("To")
-    if sent_to is not None:
-        sent_to = [recipient.strip() for recipient in sent_to.split(",")]
+    sent_from = parse_recipients(header_dict.get("From"))
+    sent_to = parse_recipients(header_dict.get("To"))
+    bcc_recipient = parse_recipients(header_dict.get("Bcc"))
+    cc_recipient = parse_recipients(header_dict.get("Cc"))
+    message_id = header_dict.get("Message-ID")
+    if message_id:
+        message_id = _strip_angle_brackets(message_id)
 
     element_metadata = ElementMetadata(
+        bcc_recipient=bcc_recipient,
+        cc_recipient=cc_recipient,
+        message_id=message_id,
```

Comment:
@scanny - What do you think about `email_message_id` vs `message_id` for this? We also have our own ID, was thinking we might want to differentiate.

Reply:
Yeah, I'd say so. This is in metadata, right? So folks will be encountering that field when email is far from their mind, so good to qualify it I'd say. Like we do `image_base64` and `image_mime_type`.

`message_id` is already established though, right? So not sure how to go about changing something like that. I suppose we either assume no on is using it and just go for it, or we populate both for a while and remove the old one later.

Reply:
No, it was added as a metadata field in this, so there shouldn't be any problem with changing it.